devtools/maven/src/main/resources/create-extension-templates/runtime-pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>[=groupId]</groupId>
+        <artifactId>[=artifactId]-parent</artifactId>
+        <version>[=version]</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>[=artifactId]</artifactId>
+[#if nameBase?? ]    <name>[=namePrefix][=nameBase][=nameSegmentDelimiter]Runtime</name>
+[/#if]
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
```

Comment:
Sorry, when I made a comment about the quarkusVersion I meant the version of this plugin, for example, and its configuration. Here it is assumed it is inherited.

Reply:
Thanks for the explanation, now I see what you mean.

When adding a new extension into an existing tree (the case the current PR implements) I think that assuming the inheritance is the right thing. I guess no sane project wants to repeat the config and version in every of their multiple extension POMs. Documenting it in the mojo JavaDoc is a good idea and I am going to do that.

It is a bit unclear to me how the standalone case (that the current PR is not implementing) should look like. I think regardless whether the standalone scenario will produce a source tree capable of hosting a single or multiple extensions, it will always have some sort of a parent pom where these things can (and should) be set.

Having said that, I believe there is no need to add mojo options for configs/versions of maven plugins. All we need is to document our opinionatedness. WDYT?

Reply:
In general, I disagree. If the intention is to provide a tool that will have set the basic project up, meaning to start a new project, then for users who actually start from nothing the current implementation will not go all the way, which surely will be disappointing. I'm not blaming you for not solving all the issues, of course, your contribution is very much appreciated. I am just saying that, imo, it's incomplete.
What it could do in addition, is check whether it is being executed in the context of an existing project and whether the necessary plugin configs can be inherited from the parent POM. If they can't be found then it should add the plugin configs to the POM to be generated.

Reply:
> What it could do in addition, is check whether it is being executed in the context of an existing project and whether the necessary plugin configs can be inherited from the parent POM. If they can't be found then it should add the plugin configs to the POM to be generated.

I must say the "necessary" is a bit unclear to me. I wonder how the set of "necessary plugin configs" could be defined? Will such a definition stay stable over time? I wonder whether it is possible to implement this in a maintainable way. I am spontaneously skeptic about it but maybe the "necessary configs" are actually easy to define. Could you please give a hint about what is "necessary"?

Reply:
Yes, that would actually be the version and config of the quarkus-bootstrap-maven-plugin. As to the stability, the project will generated for a specific version of Quarkus, so it will have to be compatible with that version, of course. It doesn't have to be compatible with older or newer Quarkus versions.

Reply:
resolved in 52ad079